tree-sitter/tree-sitter.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.4
 name:                tree-sitter
-version:             0.4.0.0
+version:             0.5.0.0
```

Comment:
I think we shouldn’t be re-versioning in this PR, since that’ll have conflicts once we release the stuff on `master` & in #225 (which we’ll probably want to happen before this PR).